assemblers/org.osbuild.error

```diff
@@ -11,7 +11,6 @@ import sys
 
 import osbuild.api
 
-
 SCHEMA = """
```

Comment:
Doesn't pep8 require two empty lines between imports and globals? Weird, but lets not fight the tools.